### PR TITLE
Added PHYS211 to Proficiency Exams

### PR DESCRIPTION
**Pull request by V-001**

### CODE REVIEW

File: docs/Proficiency Exams/PHYS211.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# PHYS211
+
+[PHYS211 (University Physics: Mechanics)](../Course%20Wiki/CS%20Course%20Offerings/PHYS211.md) is the first required physics course in the Electrical and Computer Engineering curriculum, and covers the fundementals of calculus based mechanical physics.
+
+## Who Should Take the Exam
+
+If you have a solid bakcground in Physics Mechanics at the level of at least AP Physics C, then you should consider taking the proficiency exam for PHYS221. ECE students (and most engineering students) typically take PHYS211 within their first two semesters, with PHYS211 being a prerequisite for [PHYS212 (University Physics: Elec & Mag)](../Course%20Wiki/CS%20Course%20Offerings/PHYS212.md) and a requirement for Junior Eligibility.
```
Comment (aidanrodgers) [inline docs/Proficiency Exams/PHYS211.md]: Typo on "background".  PHYS221 -> PHYS211.

File: docs/Proficiency Exams/PHYS211.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# PHYS211
+
+[PHYS211 (University Physics: Mechanics)](../Course%20Wiki/CS%20Course%20Offerings/PHYS211.md) is the first required physics course in the Electrical and Computer Engineering curriculum, and covers the fundementals of calculus based mechanical physics.
```
Comment (aidanrodgers) [inline docs/Proficiency Exams/PHYS211.md]: mechanical physics -> mechanics, typo on "fundamentals"

File: docs/Proficiency Exams/PHYS211.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- Simple Harmonmic Motion
+- Simple and Physical Pendula
+- Harmonic Waves
+- Waves and Superposition
+- Fluid Statics
+- Fluid Dynamics
+
+## Exam Format
+
+The exam consists of both multiple choice and fill in the blank questions given through Prairietest. Typically, you are provided a [formula sheet](https://courses.physics.illinois.edu/phys211/fa2023/formula_sheet.pdf) that can be accessed through the prairietest exam. The exam may cover all content in the course and is also used as a makeup final exam for students that have an IC in the class from the previous semester.
```
Comment (aidanrodgers) [inline docs/Proficiency Exams/PHYS211.md]: explain what IC stands for

File: docs/Proficiency Exams/PHYS211.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+## How to Prepare
+
+The profiency exam (in recent past) has been similar to the final exam for this course. Focus on making sure you understand the fundementals of the concepts and use the practice exams avaiable through the previous semester's course website.
```
Comment (aidanrodgers) [inline docs/Proficiency Exams/PHYS211.md]: typo on "proficiency", "available".  Change "fundamentals of the concepts" to "fundamental concepts"

File: docs/Proficiency Exams/PHYS211.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## Exam Format
+
+The exam consists of both multiple choice and fill in the blank questions given through Prairietest. Typically, you are provided a [formula sheet](https://courses.physics.illinois.edu/phys211/fa2023/formula_sheet.pdf) that can be accessed through the prairietest exam. The exam may cover all content in the course and is also used as a makeup final exam for students that have an IC in the class from the previous semester.
+
+## How to Prepare
+
+The profiency exam (in recent past) has been similar to the final exam for this course. Focus on making sure you understand the fundementals of the concepts and use the practice exams avaiable through the previous semester's course website. 
+
+Practice exams for previous semester's finals should be publically available through the course website for PHYS211. You should be able to access this through the physics department website under the [PHYS211 listing](https://physics.illinois.edu/academics/courses/PHYS211) in their course catalog.
```
Comment (aidanrodgers) [inline docs/Proficiency Exams/PHYS211.md]: semester's -> semesters', publically -> publicly.

File: docs/Proficiency Exams/PHYS211.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+## Life After
+
+If you pass the PHYS211 proficiency exam, congrats! You have a fundemental understanding of mechancial physics and are ready to tackle the next physics challenge that comes your way.
```
Comment (aidanrodgers) [inline docs/Proficiency Exams/PHYS211.md]: typo on "fundamental", "mechancial physics" -> "mechanics".  Mention PHYS212.

**Review by aidanrodgers — CHANGES_REQUESTED** [review]
Looks good, just some typos/small changes.

**Review by rzig — APPROVED** [review]

**Review by aidanrodgers — APPROVED** [review]

**Pull request merged**